packages/wonder-blocks-testing/src/harness/adapters/__tests__/data.test.js

```diff
@@ -10,30 +9,10 @@ describe("WonderBlocksData.adapter", () => {
         const children = <div>CONTENT</div>;
 
         // Act
-        const wrapper = mount(Data.adapter(children, []), {
-            includeDefaultTestHarness: false,
-        });
+        render(Data.adapter(children, []));
 
         // Assert
-        expect(wrapper).toHaveHTML("<div>CONTENT</div>");
-    });
-
-    it("should render children within InterceptRequests when dataIntercepts configured", () => {
```

Comment:
**note:** I don't think this test has value within the RTL context. Please let me know if you think otherwise.

Reply:
**suggestion:** I think it is important to test this still as we want to make sure that intercepts work within this component.

Perhaps the children need to make a request and render the result, showing the request was intercepted.

Reply:
I added back this test and used `useCachedEffect` for that. TBH I'm not fully sure if that's the right way to use the data adapter. Let me know if there's a better approach for this test. Happy to make the change if you think so :).